docs/data-source-matrix.md

```diff
@@ -0,0 +1,54 @@
+# Data Source Matrices
```

Comment:
Consider a title that tells the reader about the content rather than the format.